packages/ui-extensions/src/surfaces/admin/components/Image.d.ts

```diff
@@ -50,37 +50,37 @@ export type BoxBorderStyles = Extract<
   RequiredBoxProps['borderStyle'],
   'none' | 'solid' | 'dashed' | 'auto'
 >;
+export type ResponsiveBoxProps = MakeResponsivePick<
+  RequiredBoxProps,
+  | 'padding'
+  | 'paddingBlock'
+  | 'paddingBlockStart'
+  | 'paddingBlockEnd'
+  | 'paddingInline'
+  | 'paddingInlineStart'
+  | 'paddingInlineEnd'
+>;
```

Comment:
This seems like a possible side effect. Can you tophat and see if there's some weird padding coming out in image docs? Might be worth digging into because image should only be accessing border props off box